lib/Api/SilaApi.php

```diff
@@ -292,6 +330,142 @@ public function linkAccount(
+        );
+        $path = "/update_account";
+        $json = $this->serializer->serialize($body, 'json');
+        $headers = [
+            self::AUTH_SIGNATURE => EcdsaUtil::sign($json, $this->configuration->getPrivateKey()),
+            self::USER_SIGNATURE => EcdsaUtil::sign($json, $userPrivateKey)
+        ];
+        $response = $this->configuration->getApiClient()->callApi($path, $json, $headers);
+        return $this->prepareResponse($response, UpdateAccountResponse::class);
+    }
+
+    /**
+     * Verify KYC status of an entity in other application.
+     *
+     * @param string $userHandle
+     * @param string $queryAppHandle
+     * @param string $queryUserHandle
+     * @return ApiResponse
+     */
+     public function checkPartnerKYC(
+        string $userHandle,
+        string $queryAppHandle,
+        string $queryUserHandle
+    ): ApiResponse {
+        $body = new CheckPartnerKYCMessage(
+            $userHandle,
+            $this->configuration->getAppHandle(),
+            $queryAppHandle,
+            $queryUserHandle
+        );
+        $path = "/check_partner_kyc";
+        $json = $this->serializer->serialize($body, 'json');
+        $headers = [
+            self::AUTH_SIGNATURE => EcdsaUtil::sign($json, $this->configuration->getPrivateKey())
+        ];
+        $response = $this->configuration->getApiClient()->callApi($path, $json, $headers);
+        return $this->prepareResponse($response);
+    }
+
+    /**
+     * Verify if end-user meets the risk requirements for instant ACH.
+     *
+     * @param string $userHandle
+     * @param string $userPrivateKey
+     * @param string|null $accountName
+     * @return ApiResponse
+     */
+     public function checkInstantACH(
+        string $userHandle,
+        string $userPrivateKey,
+        string $accountName = null
+    ): ApiResponse {
+        $body = new checkInstantACHMessage(
+            $userHandle,
+            $this->configuration->getAppHandle(),
+            $accountName
+        );
+        $path = "/check_instant_ach";
+        $json = $this->serializer->serialize($body, 'json');
+        $headers = [
+            self::AUTH_SIGNATURE => EcdsaUtil::sign($json, $this->configuration->getPrivateKey()),
+            self::USER_SIGNATURE => EcdsaUtil::sign($json, $userPrivateKey)
+        ];
+        $response = $this->configuration->getApiClient()->callApi($path, $json, $headers);
+        return $this->prepareResponse($response, BaseResponse::class);
+    }
+
+    /**
+     * Uses an account name to delete a bank account from an entity.
+     *
+     * @param string $userHandle
+     * @param string $userPrivateKey
+     * @param string $accountName
+     * @return ApiResponse
+     */
+    public function deleteAccount(
+        string $userHandle,
+        string $accountName
+    ): ApiResponse {
+        $body = new DeleteAccountMessage(
+            $userHandle,
+            $this->configuration->getAppHandle(),
+            $accountName,
```

Comment:
@AlferOro For PHP 7.2 the trailing comma is not enabled. Please remove this on the method call.